cli/azd/internal/repository/infra_confirm.go

```diff
@@ -33,7 +33,7 @@ func (i *Initializer) infraSpecFromDetect(
 
 	dbPrompt:
 		for {
-			dbName, err := i.promptDbName(ctx, database)
+			dbName, err := promptDbName(i.console, ctx, database)
```

Comment:
This change builds on top of #4508, `infra_confirm.go` is unchanged between these two PRs, so the three files that need reviewed are:

- app_init.go
- app_init_test.go
- next-steps-alpha.mdt